readme.md

```diff
@@ -902,4 +902,4 @@ You can find some examples in the [TypeScript docs](https://www.typescriptlang.o
 
 ## License
 
-SPDX-License-Identifier: (MIT OR CC0-1.0)
+SPDX-License-Identifier: (MIT OR CC0-1.0)
```

Comment:
Don't make unrelated changes.

Reply:
I believe the trailing new line was changed.

Reply:
@orimiles5 This still needs to be fixed.